pom.xml

```diff
@@ -125,8 +125,7 @@
     <org.json.version>20070829</org.json.version>
     <org.juzu.version>1.1.0-M1</org.juzu.version>
     <org.mockito.version>1.8.5</org.mockito.version>
-    <org.mortbay.jetty.cometd-bayeux.version>6.1.11</org.mortbay.jetty.cometd-bayeux.version>
-    <org.mortbay.jetty.version>0.9.20080221</org.mortbay.jetty.version>
+    <org.cometd.version>3.0.5</org.cometd.version>
```

Comment:
All properties are alphabetically sorted.
Can you add this property just before the <org.easymock.version> property please? Thanks